shell/packages/sandstorm-ui-autocomplete-input/autocomplete-client.js

```diff
@@ -0,0 +1,210 @@
+window.UserContacts = new Mongo.Collection("userContacts");
+
+Template.contactInputBox.onCreated(function () {
+  var self = this;
+  this.currentText = new ReactiveVar(null);
+  this.inputActive = new ReactiveVar(false);
+  this.selectedContacts = this.data.contacts;
+  this.selectedContactsIds = new ReactiveVar([]);
+  this.highlightedContact = new ReactiveVar({_id: null});
+  this.subscribe("userContacts");
```

Comment:
Oh, interesting. I had assumed that the Contacts subscription would be a one-time global subscription. I suppose one advantage to your approach here is that the non-reactivity of the subscription matters less; if one your contacts uploads a different profile picture, you'll see it next time you open a new contactInputBox.